CuratedContent/WorkingEffectivelyWithLegacyCode.md

```diff
@@ -0,0 +1,25 @@
+# Working Effectively with Legacy Code
+
+## Feathers, Michael, 2005
```

Comment:
Need isbn

Reply:
@prwolfe,

Looks like if we add the ISBN to this document, it will be the first:

```
$ cd betterscientificsoftware.github.io
$ find . -type f -exec grep -nHi isbn {} \;
[empty]
```

Therefore, I don't know where to put it.  But I will put it somewhere.

Reply:
Interesting - we should ask if there is a reason not to. It just seemed like the right thing to my mind.